frontend/src/App.tsx

```diff
@@ -15,6 +16,8 @@ export default function App() {
                     />}/>
                     <Route path="/Login" element={<LoginPage
                     />}/>
+                    <Route path="/MainPage" element={<MainPage
```

Comment:
```suggestion
                    <Route path="/" element={<MainPage
```
Would make your Main Page the default Page.